modules/enabling-external-ip-collection-central.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * operating/visualizing-external-entities.adoc
+:_module-type: PROCEDURE
+[id="enabling-external-ip-collection-central_{context}"]
+= Enabling external IP collection in Central
+
+[role="_abstract"]
+There are two environmental variables that control the external IP collection in Central `ROX_EXTERNAL_IPS` and `ROX_NETWORK_GRAPH_EXTERNAL_IPS`.
+
+* To only view the external entities in the Network Graph, enable the `ROX_NETWORK_GRAPH_EXTERNAL_IPS` environmental variable.
+* To view external entities in the Network Graph and query external entities by using the API, enable the `ROX_EXTERNAL_IPS` environmental variable.
```

Comment:
This is not the way it works: `ROX_EXTERNAL_IPS` is also required to get external-IPs displayed in the Network Graph.

To be more precise, `ROX_EXTERNAL_IPS` is a general feature toggle for everything related to external-IPs in Central. Once this is set, then `ROX_NETWORK_GRAPH_EXTERNAL_IPS` can be used to display those external-IPs.